defaults/main.yaml

```diff
@@ -161,3 +161,69 @@ zookeeper_services:
 ## Ensure our topic does no longer exist if it is in the topic list
 #  - name: unused_topic
 #    state: absent
+
+#Log4J Vars
+log4j_logDir: "{{ dst_path }}/confluent-{{ confluent_version }}/logs"
```

Comment:
Sorry for nitpicking about this. Could you change the variable names to snake case? To follow the Ansible code style

Reply:
Yeah, you're right. I changed all variables to snake case